### ISSUE TITLE
"SyntaxError: Use of const in strict mode" while installing on Travis (for node@0.12)

### ISSUE THREAD

**Issue by broofa** [typicode/husky#308]
In trying to add support for enforcing Conventional Commits on the `uuid` module I ran into the issue, below.  This is happening on Travis, while running the unit tests for node@0.12.  The tests for more recent versions of node work.

----
```
$ git clone --depth=50 --branch=conventional_commits https://github.com/kelektiv/node-uuid.git kelektiv/node-uuid
Cloning into 'kelektiv/node-uuid'...
remote: Counting objects: 374, done.
remote: Compressing objects: 100% (186/186), done.
remote: Total 374 (delta 198), reused 340 (delta 177), pack-reused 0
Receiving objects: 100% (374/374), 124.10 KiB | 15.51 MiB/s, done.
Resolving deltas: 100% (198/198), done.
$ cd kelektiv/node-uuid
$ git checkout -qf 4d3ae93ec18647ea88b454c7b8d3db912a25446f
$ export PATH=./node_modules/.bin:$PATH
Updating nvm
nvm.install
2.90s$ nvm install 0.12
Downloading and installing node v0.12.18...
Downloading https://nodejs.org/dist/v0.12.18/node-v0.12.18-linux-x64.tar.xz...
Computing checksum with sha256sum
Checksums matched!
Now using node v0.12.18 (npm v2.15.11)
$ node --version
v0.12.18
$ npm --version
2.15.11
$ nvm --version
0.33.11
63.76s$ npm install 

[... **snip** ...]

> husky@0.14.3 install /home/travis/build/kelektiv/node-uuid/node_modules/husky
> node ./bin/install.js
/home/travis/build/kelektiv/node-uuid/node_modules/husky/bin/install.js:4
const path = require('path')
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by broofa** [comment]
Since it's Travis doing the install (on their test servers), I'm not sure how much control I have over the flags used.

For the time being I've solved this on our side by dropping 0.12 from our `.travis.yml`.   I'll reopen if we need to add that back in later.

**Comment by typicode** [comment]
@broofa it should be good now with husky `v1.0.0-rc12` ;) https://travis-ci.org/typicode/please-upgrade-node/jobs/400214097#L471

And as for setting flags on Travis, it's actually quite easy (was doing this before to avoid running husky's own  `install` script https://github.com/typicode/husky/blob/v1.0.0-rc.6/.travis.yml#L7-L8)
